### PR TITLE
5569-componentSemantics

### PR DESCRIPTION
**Pull request by lzach83**
## Link to Issue
Closes: #5569 

## Description of Changes
- Changes `chain` and `chainId` to `community` and `communityId` and everything similar  
## Other Considerations

- Some files were not touched most glaring is `ProfileActivityRow` which is heavily reliant on the models /types which will be addressed in #3779 
- Due to pages being done before -- this has to be merged after #5387 in order for it to fully work as intended. I will update here when master is up to date.

### CODE REVIEW

**Review by masvelio — COMMENTED** [review]
image

Look for every usage of <User> component in the codebase and change everywhere `userChainId` to `userCommunityId`

**Comment by lzach83** [comment]
Fixed the CR comments. Thank you as always @masvelio 

#5387 should take precedence over this.

File: packages/commonwealth/client/scripts/views/pages/notifications/notification_row_components.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ export const ChainEventNotificationRow = (
     proposalType,
     (chainEvent.data as any).id,
     false,
-    chainId
+    chainId,
```
Comment (timolegros) [inline packages/commonwealth/client/scripts/views/pages/notifications/notification_row_components.tsx:89]: `communityId` -> `chainEvent.chain` is a community (not a blockchain).

**Review by timolegros — CHANGES_REQUESTED** [review]
Missing changes:
- `const chain`, `chainScope` in `CWSearchBar.tsx`
- `const chainInfo` in `SearchBarCommunityPreviewRow.tsx`
- `chainId` parameter of `useNewThreadForm` function in `useNewThreadForm.ts`

**Review by masvelio — APPROVED** [review]

**Comment by lzach83** [comment]
We're good re-review this.

**Review by timolegros — APPROVED** [review]
Fixed:
- `chainValidationSchema` should not be updated to `communityValidationSchema` because the selectable options are 'solana', 'polkadot', 'ethereum', etc all of which are chains and not communities
- `ProfileActivityRow` had a `chain` prop that wasn't cast to `communityId`. The prop cannot be updated because it is extracted from a `Thread` or `CommentWithAssociatedThread` object. Once those 2 are updated the underlying prop can be updated to `communityId` as well.
- Links for comments and threads broken in search

Tested:
- edit comment
- view user pop-up
- join a new community
- create a new thread using thread form
- view your profile and another users profile
- react on a thread and comment
- view community explore page
- view user avatars
- sign in with a variety of wallets
- Search for communities, members, and threads -> #5960 
- View aave proposals page and a specific proposal page with votes (dydx)
- View the members page
- Edit collaborators -> #5962 
- Use a poll
- View notifications and notification settings page
- View topics list dropdown
- View the user dashboard
- Update users to admins or moderators
- View compound proposals (tribe)

**Pull request merged**